src/mono/wasm/build/WasmApp.targets

```
@@ -39,13 +39,19 @@
 
       <_WasmAssemblies Remove="@(_WasmAssemblies)" />
     </ItemGroup>
+    <PropertyGroup>
+      <AOTMode>LLVMOnly</AOTMode>
+      <AOTMode Condition="'$(BuildAOTProfiled)' == 'true'">AotInterp</AOTMode>
```

We should check to see if the property is already set. This would allow it to be overridden.

```suggestion
      <AOTMode Condition="'$(AOTMode)' == '' and '$(BuildAOTProfiled)' == 'true'">AotInterp</AOTMode>
      <AOTMode Condition="'$(AOTMode)' == ''">LLVMOnly</AOTMode>
```